src/Lean/Elab/Command.lean

```diff
@@ -289,7 +289,9 @@ partial def elabCommand (stx : Syntax) : CommandElabM Unit := do
             withInfoTreeContext (mkInfoTree := mkInfoTree `no_elab stx) <|
               throwError "elaboration function for '{k}' has not been implemented"
           | elabFns => elabCommandUsing s stx elabFns
-    | _ => throwError "unexpected command"
+    | _ =>
```

Comment:
Apparently the test input makes the parser pass `"/--"` as a single atom to the elaborator. Not sure where the syntax node went...